lib/pharos/scripts/configure-etcd.sh

```diff
@@ -21,6 +21,7 @@ metadata:
   name: etcd
   namespace: kube-system
 spec:
+  priorityClassName: system-cluster-critical
```

Comment:
I think this needs to be `system-node-critical` that has the highest priority. I mean if etcd would get preempted, everything breaks. :) Although this is a static pod, so I don't think these will actually ever get preempted.